remoting/src/main/java/org/apache/rocketmq/remoting/protocol/RemotingCommand.java

```diff
@@ -61,10 +61,13 @@
     static {
         final String protocol = System.getProperty(SERIALIZE_TYPE_PROPERTY, System.getenv(SERIALIZE_TYPE_ENV));
         if (!isBlank(protocol)) {
+            byte protocolByte = -1;
             try {
-                serializeTypeConfigInThisServer = SerializeType.valueOf(protocol);
+                protocolByte = (byte) Integer.parseInt(protocol);
+                System.out.println(protocolByte);
```

Comment:
IMO, we should delete these two codes.

Reply:
The method SerializeType.valueOf() need a byte type parameter.It will get a wrong result when we put a String type parameter.

Reply:
You are right , just delete `System.out.println(protocolByte);`.